legal-api/report-templates/template-parts/notice-of-articles/headerDetails.html

```diff
@@ -0,0 +1,38 @@
+ <div class="business-details-container">
+      <div class="business-details">
+         <table class="business-details-table" role="presentation">
+            <tr>
+               <td>
+                  <table class="business-info" role="presentation">
+                     <tr>
+                        <td class="section-sub-title">Incorporation Number:</td>
+                        <td>{{business.identifier}}</td>
+                     </tr>
+                     <tr>
+                        <td class="section-sub-title">Issued Date and Time:</td>
+                        <td>{{effective_date_time}}</td>
+                     </tr>
+                     <tr>
+                        <td class="section-sub-title">Recognition Date and Time:</td>
+                        <td>{{recognition_date_time}}</td>
+                     </tr>
+                  </table>
+                  <div class="filing-identifier">
+                     Filing #{{header.filingId}}
```

Comment:
Does this new code block also need to be on COA and COD?

Also, why is this block not in the table above?

Do you have a design or sample output I could look at?

Reply:
If you dont include this you wont see the filing id and incorporation # in the footer. This is a hidden element because filing id is not used anywhere else in the report and you can look at the style to see how footer is implemented